account_financial_report/report/templates/journal_ledger.xml

```diff
@@ -155,11 +158,11 @@
                  name="taxes"/>
             <div class="act_as_cell amount"
                  name="debit">
-                <b><span t-field="journal.debit"/></b>
+                <b><span t-esc="journal['debit']"/></b>
```

Comment:
This is losing the widget format. Use `t-widget` for restoring it.

Reply:
Fixed